pkgs/build-support/setup-hooks/patch-shebangs.sh

```diff
@@ -84,19 +84,18 @@ patchShebangs() {
         fi
 
         # Strip trailing whitespace introduced when no arguments are present
-        newInterpreterLine="$(echo "$newPath $args" | sed 's/[[:space:]]*$//')"
+        newInterpreterLine=$(sed 's/[[:space:]]*$//' <<< "$newPath $args")
```

Comment:
If you want to avoid starting up a process
```suggestion
        newInterpreterLine="$newPath $args"
        newInterpreterLine=${newInterpreterLine%${newInterpreterLine##*[![:space:]]}}
```
But it may raise a few eyebrows 😉

Reply:
It's potentially called quite often, so it is worth it.